syscall/README.md

```diff
@@ -1,22 +1,22 @@
-liboeposix:
-===========
+liboesyscall
```

Comment:
Nit: we're removing the colons in some cases but not others, just pick one for consistency.

Reply:
Yeah, restored.